aptos-move/framework/aptos-framework/sources/staking_proxy.spec.move

```diff
@@ -21,23 +24,60 @@ spec aptos_framework::staking_proxy {
 
     spec set_vesting_contract_operator(owner: &signer, old_operator: address, new_operator: address) {
         // TODO: Can't verify `update_voter` in while loop.
-        pragma aborts_if_is_partial;
+        pragma verify = false;
     }
 
     spec set_staking_contract_operator(owner: &signer, old_operator: address, new_operator: address) {
-        use aptos_std::simple_map;
-        use aptos_framework::staking_contract::{Store};
+        pragma verify = false;
         // TODO: Verify timeout and can't verify `staking_contract::switch_operator`.
         pragma aborts_if_is_partial;
+        // TODO: set because of timeout (property proved)
+        pragma verify_duration_estimate = 120;
+
+        include SetStakingContractOperator;
+    }
+
+    spec schema SetStakingContractOperator {
```

Comment:
If it cannot be proved, please comment this out. Also, it would be nice if you can find which clause in this schema causes timeout